docs/Table.md

```diff
@@ -19,41 +19,68 @@ const { query } = initialize({ iotsitewiseClient });
 <Table
   viewport={{ duration: '5m' }}
   
-  tableColumns={[
+  items={[
     {
-      header: 'Tempature',
-      rows: [
-        toId({ assetId: 'engine-1', propertyId: 'tempature' }),
-        toId({ assetId: 'engine-2', propertyId: 'tempature' }),
-      ]
+      rpm: {
+        $cellRef: {
+          id: toId({ assetId: 'turbine-1', propertyId: 'rotation_per_minute' }),
```

Comment:
we should add the import statement toId to give the full example

Reply:
Import statements are the top of the code block.